app/src/routes/setup/form.vue

```diff
@@ -0,0 +1,182 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useI18n } from 'vue-i18n';
+import { useServerStore } from '@/stores/server';
+import { storeToRefs } from 'pinia';
+import { initialValues, useFormFields } from './form';
+import { SetupForm } from '@directus/types';
+
+const { t } = useI18n();
+
+const { info } = storeToRefs(useServerStore());
+
+const props = withDefaults(
+	defineProps<{
+		register?: boolean;
+		skipLicense?: boolean;
+		modelValue?: SetupForm;
+		errors?: Record<string, any>[];
+		utmLocation?: string;
+	}>(),
+	{
+		register: true,
+		skipLicense: false,
+		utmLocation: '',
+		modelValue: () => initialValues,
+	},
+);
+
+const value = defineModel<SetupForm>();
+
+const license = computed({
+	get: () => value.value?.license ?? false,
+	set: (val: boolean) => {
+		if (value.value) {
+			value.value.license = val;
+		}
+	},
+});
+
+const product_updates = computed({
+	get: () => value.value?.product_updates ?? false,
+	set: (val: boolean) => {
+		if (value.value) {
+			value.value.product_updates = val;
+		}
+	},
+});
+
+const fields = useFormFields(props.register, value);
+</script>
+
+<template>
+	<div class="setup-form" :class="{ skipLicense }">
+		<template v-if="register">
+			<h1>{{ t('setup_welcome') }}</h1>
+			<p>{{ t('setup_info') }}</p>
+		</template>
+		<v-form
+			v-model="value"
+			:initial-values="initialValues"
+			:validation-errors="errors"
+			:show-validation-errors="false"
+			:fields="fields"
+			disabled-menu
+		></v-form>
+		<v-notice>
+			<span v-md="t('setup_license_notice')"></span>
+			<br />
+			<i18n-t keypath="setup_license_follow_up" tag="span">
+				<template #contactOurTeam>
+					<a
+						:href="`https://directus.io/license-request?utm_source=self_hosted&utm_medium=product&utm_campaign=2025_10_kyc&utm_term=${info.version}&utm_content=${utmLocation}_contact_our_team_link`"
```

Comment:
Does `info.version` exist here before login?

Reply:
Great idea, I'll do that!